src/styles/container.scss

```diff
@@ -0,0 +1,23 @@
+.container {
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  padding: 50px 40px;
+  justify-content: space-between;
+  gap: 46px;
+
+  @media (min-width: 1200px) {
+    padding-inline: 128px;
+  }
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 200px);
+  }
```

Comment:
This media query is correctly set to change the grid-template-columns property to repeat(4, 200px) when the width is 1024px or more. This will create 4 columns of 200px each. However, make sure that the media queries are ordered correctly.